src/scripts/dataExchange/dataExchange.ts

```diff
@@ -212,3 +185,51 @@ class DataExchange {
 
 };
 module.exports = new DataExchange();
+
+async function readImportParams(config: Record<string, any>, importParams: any) {
+    // import
+    // read folders, and subfolders if directory, otherwise read file
+    const params = Object.keys(importParams);
+    for (let i = 0; i < params.length; i++) {
+      const param = params[i];
+      // adjust file path if relative
+      const fileOrDir = importExportDataUtil.getFileAbsPath(importParams[param]);
+      // @ts-ignore
+      const isFileReadable = await importExportDataUtil.isReadable(fileOrDir);
+      if (isFileReadable) {
+        // @ts-ignore
+        await importExportDataUtil.readData(fileOrDir, [param], config);
+      } else {
+        delete importParams[param];
+        console.warn(
+          `"${fileOrDir}" does not exist or is not readable. Please check path or permissions.`
+        );
+      }
+    } 
+}
+
+async function readExportParams(config: Record<string, any>, exportParams: any) {
+  if (!exportParams) {
```

Comment:
check is redundant: https://github.com/SAP/wdio-qmate-service/pull/277/files?diff=split&w=0#diff-bb5bafb01127841195d1c3c05a6ff59f0eb078f7ec1cb36892c2f179b6a5491dR38

Reply:
removed the check